include/sot/core/device.hh

```diff
@@ -40,15 +40,15 @@ namespace dynamicgraph {
     /// Define the type of input expected by the robot
     enum ControlInput
     {
-      CONTROL_INPUT_POSITION=0,
-      CONTROL_INPUT_VELOCITY=1,
-      CONTROL_INPUT_ACCELERATION=2,
+      CONTROL_INPUT_NO_INTEGRATION=0,
+      CONTROL_INPUT_ONE_INTEGRATION=1,
```

Comment:
The use of ControlInput in the device class is in the increment and integrate functions to get the state and velocity signals from the input control signal. The CONTROL_TYPE_POSITION is the only type where the velocity signal is not being filled from the control signal or its integral.

For the torque control mode, we need to not integrate the input control signal. We are using the name CONTROL_INPUT_POSITION for that, which is misleading.